tests/console/pam.pm

```diff
@@ -36,9 +36,6 @@ sub run {
         zypper_ar("$qa_head_repo", name => 'qa-head-repo');
     }
     zypper_call('install bats pam-test pam pam-config snapper perl');
-    if (is_tumbleweed()) {
-        zypper_call('install pam-deprecated "group(wheel)"');
```

Comment:
group(wheel) should probably remain - as we still have the test:

```
@test "modify group access" {                                         # case 17
    echo "*;*;$USER_NOR;Al0000-2400;wheel" >> /etc/security/group.conf
```

Reply:
> Looks almost identical to #13307 :)

Oh indeed! Hm, what I see in that in sle, the tests don't fail yet, as  the pam version is older than the one that these 2 modules are being deprecated (eg for sle15sp3, I see `pam-1.3.0-6.50.1.x86_64` ). So I presume that we will see the same issues there and consequently, the logic of skipping will make no sense anymore.